modules/lead/toolchain/grafeas.tf

```diff
@@ -18,7 +18,7 @@ module "certificate" {
   enabled = true
   name = "grafeas-cert"
   namespace = var.namespace
-  domain = var.root_zone_name
+  domain = "grafeas-server.toolchain.svc"
```

Comment:
The namespace name `toolchain` is configurable via `var.namespace`...can you use that instead?

Reply:
Fixed to use namespace var.